integrations/openai.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: OpenAI
+---
+
+<Note>
+  We support the OpenAI python library starting at version 1.0.0.
+</Note>
+
+The benefits of this integration is that you can see the OpenAI API calls in a step in the UI, and you can explore them in the prompt playground.
+
+You need to add `cl.instrument_openai()` after creating your OpenAI client.
+
+<Warning>
+  You shouldn't configure this integration if you're already using another integration like Embedchain, Haystack, Langchain or LlamaIndex. Both integrations would record the same generation and create duplicate steps in the UI.
+</Warning>
+
+## Prerequisites
+
+Before getting started, make sure you have the following:
+
+- A working installation of Chainlit
+- The OpenAI package installed
+- An OpenAI API key
+- Basic understanding of Python programming
+
+## Step 1: Create a Python file
+
+Create a new Python file named `app.py` in your project directory. This file will contain the main logic for your LLM application.
+
+## Step 2: Write the Application Logic
+
+In `app.py`, import the necessary packages and define one function to handle messages incoming from the UI.
+
+```python
+from openai import AsyncOpenAI
+
+from chainlit.playground.providers import ChatOpenAI
```

Comment:
dead import i think

Reply:
Good point, I've removed it.

It makes me thing the integration is tagging everything as `openai` vs the manual tagging that could be things like `openai-chat`, `azure-openai-chat`...